forge-gui/res/cardsfolder/upcoming/open_the_way.txt

```diff
@@ -0,0 +1,6 @@
+Name:Open the Way
+ManaCost:X G G
+Types:Sorcery
+A:SP$ DigUntil | Amount$ X | Cost$ X G G | Valid$ Card.Land | ValidDescription$ land | FoundDestination$ Battlefield | Tapped$ True | RevealedDestination$ Library | RevealedLibraryPosition$ -1 | RevealRandomOrder$ True | XMaxLimit$ X | SpellDescription$ X can't be greater than the number of players in the game. Reveal cards from the top of your library until you reveal X land cards. Put those land cards onto the battlefield tapped and the rest on the bottom of your library in a random order.
+SVar:X:PlayerCountPlayers$Amount
```

Comment:
doesn't X need to be xPaid ?

Reply:
Hmm I've used other cards as a template and it seems to work in testing

Reply:
Oh I see what you mean